### DISCUSSION TITLE
why is storybook still dependent on webpack 4?

### DISCUSSION THREAD

**Discussion by blainebell** [storybookjs/storybook#14947] [General] [answered]
I am having problems upgrading to webpack 5, and was wondering if its possible.  thanks!

https://github.com/storybookjs/storybook/blob/next/package.json#L269

**Comment by shilman** [comment, marked as answer]
Did you read this?

https://gist.github.com/shilman/8856ea1786dcd247139b47b270912324

**Reply by blainebell** [reply]
Thanks a lot for the quick response, Michael! Much appreciated.  Looks like that's exactly what I need.  I will try that.  You can close this issue.

**Reply by blainebell** [reply]
actually, since this fix is dependent on a prerelease, do you know when the actual next release is going to happen?  thanks!

**Reply by shilman** [reply]
Experimental webpack5 support is part of 6.2 stable; the gist was written when it was a prerelease.
